### ISSUE TITLE
Uncaught TypeError: this.getActivePaneContainer(...).activateNextPane is not a function

### ISSUE THREAD

**Issue by AgentAntelope** [atom/atom#14442]
[Enter steps to reproduce:]

1. Open a dock (I used the sidebar, but this works in any dock)
2. Trigger `window:focus-next-pane`

expected:
EITHER
1) Next pane/dock is selected
2) Nothing happens, as dock only has one pane

actual:
The below error happens.

**Atom**: 1.17.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.4
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: this.getActivePaneContainer(...).activateNextPane is not a function

```
At /Applications/Atom.app/Contents/Resources/app/src/workspace.js:1559

TypeError: this.getActivePaneContainer(...).activateNextPane is not a function
    at Workspace.activateNextPane (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:1559:1)
    at atom-workspace.window:focus-next-pane (/Applications/Atom.app/Contents/Resources/app/src/register-default-commands.js:168:1)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app/src/command-registry.js:265:1)
    at /Applications/Atom.app/Contents/Resources/app/src/command-registry.js:3:1
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:610:1)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app/node_modules/atom-keymap/lib/keymap-manager.js:401:1)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:100:1)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/window-event-handler.js:3:1)
```

### Commands

```
  2x -0:51.1.0 window:toggle-left-dock (input.hidden-input)
  3x -0:45.6.0 pane:show-previous-item (input.hidden-input)
  6x -0:40.7.0 pane:show-next-item (input.hidden-input)
  5x -0:39.5.0 pane:show-previous-item (input.hidden-input)
     -0:38.4.0 pane:show-next-item (input.hidden-input)
     -0:36.3.0 window:toggle-left-dock (input.hidden-input)
     -0:33.7.0 core:move-right (input.hidden-input)
     -0:33.7.0 core:move-down (input.hidden-input)
     -0:33 core:move-left (input.hidden-input)
     -0:30.6.0 command-palette:toggle (input.hidden-input)
     -0:28 core:backspace (input.hidden-input)
     -0:26.6.0 editor:consolidate-selections (input.hidden-input)
     -0:26.6.0 core:cancel (input.hidden-input)
     -0:25.5.0 window:focus-next-pane (input.hidden-input)
  4x -0:24.4.0 core:move-down (input.hidden-input)
  3x -0:18.8.0 window:focus-next-pane (div.tool-panel.tree-view)
```

### Non-Core Packages

```
auto-update-packages 1.0.1 
autocomplete-haskell 0.7.2 
busy-signal 1.4.1 
file-icons 2.1.5 
fonts 2.1.0 
git-blame 1.0.2 
intentions 1.1.2 
jslint 1.5.1 
language-babel 2.61.0 
language-gherkin 1.0.4 
language-haskell 1.12.1 
linter 2.1.4 
linter-eslint 8.1.7 
linter-ui-default 1.5.0 
lisp-paredit 0.5.5 
nyancat 0.14.0 
sort-lines 0.14.0 
swackets 0.26.0 
```

**Comment by 50Wliu** [comment]
See also atom/command-palette#79.

**Comment by jasonrudolph** [comment]
I've cherry-picked this fix onto the 1.17-releases branch (https://github.com/atom/atom/commit/0094050c628d042ff4a569bb7ac4602fbabbbae3) and the 1.18-releases branch (768afd6bba4e42d44fe973d07ed15dd8e479ba67). This fix will be included in the next  1.17 patch release and 1.18-beta patch release.
